src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/MermaidKernelExtensionTests.cs

```diff
@@ -123,7 +172,24 @@ public void can_generate_class_diagram_from_generic_type()
         [Fact]
         public void can_generate_class_diagram_to_a_specified_depth()
         {
-            var diagram = typeof(List<Dictionary<string, object>>).ToClassDiagram(graphDepth:2);
+            var diagram = typeof(List<Dictionary<string, object>>).ToClassDiagram(new ClassDiagramConfiguration(2));
+
+            this.Assent(diagram.ToString());
```

Comment:
I would do this without `Assent`. Maybe verify that a specific 3rd-level item isn't present?

Reply:
that means parsing lines :(

Reply:
I made it